app/models/label.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Resposible for storing configurations
+class Label < ApplicationRecord
+  belongs_to :account
+
+  enum scope: { general: 0, restricted: 1 }
+
+  validates :scope,
+            inclusion: { in: scopes.keys }
+
+  validates :key,
+            length: 3..255,
+            format: { with: /\A[a-z0-9_-]+\z/ },
+            uniqueness: { scope: :account_id }
+
+  validates :value,
```

Comment:
We need format validation here `format: { with: /\A[A-Za-z0-9_-]+\z/ }`

Reply:
thx, done